lifecycle-operator/controllers/options/keptnconfig_controller.go

```diff
@@ -40,19 +44,67 @@ type KeptnConfigReconciler struct {
 	Log             logr.Logger
 	LastAppliedSpec *optionsv1alpha1.KeptnConfigSpec
 	config          config.IConfig
+	Namespace       string
 }
 
-func NewReconciler(client client.Client, scheme *runtime.Scheme, log logr.Logger) *KeptnConfigReconciler {
+func NewReconciler(client client.Client, scheme *runtime.Scheme, log logr.Logger, namespace string) *KeptnConfigReconciler {
 	return &KeptnConfigReconciler{
-		Client: client,
-		Scheme: scheme,
-		Log:    log,
-		config: config.Instance(),
+		Client:    client,
+		Scheme:    scheme,
+		Log:       log,
+		config:    config.Instance(),
+		Namespace: namespace,
+	}
+}
+
+// variables needed for Keptn Gateway
+const gatewayPort = 8080
+
+func getKeptnGatewayDeployment(namespace string) *appsv1.Deployment {
+	return &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "keptn-gateway",
+			Namespace: namespace,
+		},
+		Spec: appsv1.DeploymentSpec{
+			Template: corev1.PodTemplateSpec{
+				Spec: corev1.PodSpec{
```

Comment:
please add the recommended labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/